src/components/implementation/no_interface/cFE_booter/cFE_emu_support.c

```diff
@@ -0,0 +1,117 @@
+#include <stdio.h>
+
+#include <cos_component.h>
+#include <cos_debug.h>
+#include <cos_types.h>
+
+#include <memmgr.h>
+
+#include <cfe_error.h>
+
+#include <cFE_emu.h>
+
+union shared_region *shared_regions[16];
+
+int
+emu_backend_request_memory(spdid_t client)
```

Comment:
should it be `cFE_emu`?

Reply:
We're already in the `cFE` component. No need to namespace unnecessarily IMHO. I'm open to changing this tho.